tests/mic/test_audio_input.py

```diff
@@ -0,0 +1,23 @@
+"""
+Mock tests for microphone input
+"""
+from unittest.mock import patch
+
+from spokestack.mic.pyaudio import PyAudioMicrophoneInput
+
+
+@patch("spokestack.mic.pyaudio.pyaudio")
+def test_audio_input(mock_class):
+    """ Test for microphone input """
+    config = {"sample_rate": 16000, "frame_width": 10}
+    mic = PyAudioMicrophoneInput(**config)
```

Comment:
instead of passing a dictionary, it would be simpler just to pass the kwargs directly here